test/draw.py

```diff
@@ -56,49 +33,68 @@ class TestDraw(unittest.TestCase):
+
+        ''' This function is to create an empty image with a specific dimension
+            with white background, and black/white colored '''
+
+        image, canvas = get_image('L', (15,90),'white')
+
+        for i in range(len(draw_points) - 1):
+            draw(canvas, (draw_points[i + 0], draw_points[i + 1]), 'A')
 
-        fileName = sys._getframe().f_code.co_name + '.png'
+        """ saving the file and closing it """
 
-        image.save(outputPath+"/"+fileName, fileType)
+        image.save(result_file)
         image.close()
 
-        resultFile = outputPath + "/" + fileName
-        referenceFile = referencePath + "/" + fileName
+        """ validate the resultant file against the reference images"""
 
-        # compare results agains reference data
-        with open(resultFile, "rb") as result:
-            with open(referenceFile, "rb") as reference:
-                self.assertTrue(reference.read() == result.read())
+        validate(reference_file, result_file)
 
     # Result image for drawWire
     def test_drawWire(self):
-        image = Image.new(imageMode, imageSize, imageBackground)
-        canvas = ImageDraw.Draw(image)
 
-        drawWire(canvas, points)
+        """ Create the file name and its saving path, and specify the reference file to compare to."""
 
-        fileName = sys._getframe().f_code.co_name + '.png'
+        image_name = filename(sys._getframe().f_code.co_name)
+        result_file, reference_file = get_path(image_name)
 
-        image.save(outputPath + "/" + fileName, fileType)
+        ''' This function is to create an empty image with a specific dimension
+            with white background, and black/white colored '''
+
+        image, canvas = get_image('L',(640,480),'white')
+
+        drawWire(canvas, regularPolygon(3, np.array([160, 120]), 50))
+        drawWire(canvas, regularPolygon(4, np.array([480, 120]), 90))
+        drawWire(canvas, regularPolygon(5, np.array([420, 360]), 60))
+        drawWire(canvas, regularPolygon(6, np.array([160, 360]), 80))
+        drawWire(canvas, regularPolygon(7, np.array([320, 160]), 70))
+
+        """ saving the file and closing it """
+
+        image.save(result_file)
         image.close()
 
-        resultFile = outputPath + "/" + fileName
-        referenceFile = referencePath + "/" + fileName
+        """ validate the resultant file against the reference images"""
 
-        # compare results agains reference data
-        with open(resultFile, "rb") as result:
-            with open(referenceFile, "rb") as reference:
-                self.assertTrue(reference.read() == result.read())
+        validate(reference_file, result_file)
 
     # Result image for drawSolid
     def test_drawSolid(self):
-        image = Image.new('RGB', imageSize, imageBackground)
-        canvas = ImageDraw.Draw(image)
+
+        """ Create the file name and its saving path, and specify the reference file to compare to."""
```

Comment:
function help string format should have the """ separator alone on it's own line:

"""
This function ...
"""